doc/auth-handlers.md

```diff
@@ -0,0 +1,34 @@
+## AuthImplementations
+
+If we need to enable auth handlers in Pravega controller, modify the Pravega manifest and ensure that following fields are present in the yaml file.
+
+```
+authImplementations:
+  mountPath: "/opt/pravega/pluginlib"
```

Comment:
One thing I was wondering about mountPath.   SHOULD it be something the user provides?   What I mean is, if they change it, things will actually not work (because I think the controller looks specifically in that location).    If that's true, then it shouldn't be configurable at all I feel.

Reply:
I felt that making it  configurable is better. If the path in controller changes later, from operator side no changes will be required

Reply:
got it.